### PR TITLE
WIP: HTTPS and other minor updates for config/edition (do not merge)

### PR DESCRIPTION
**Pull request by tfmorris**
This addresses some of the edits in #725, but does not include any of the proposed new entries (which are many).

### CODE REVIEW

File: openlibrary/plugins/openlibrary/pages/config_edition.page [hunk 1]
```diff
@@ -159,8 +159,8 @@
             "label": "Hathi Trust"
         },
         {
-            "url": "http://discovery.lib.harvard.edu/?itemid=|library/m/aleph|@@@",
-            "website": "http://harvard.edu",
+            "url": "https://hollis.harvard.edu/primo_library/libweb/action/display.do?doc=HVD_ALEPH@@@",
```
Comment (LeadSongDog) [inline openlibrary/plugins/openlibrary/pages/config_edition.page]: Harvard introduces a new Hollis on 12 March, so maybe hold off on this.  It seems from the preview that its permalinks will not support https, but that may be transitional. Form will be http://id.lib.harvard.edu/aleph/@@@/catalog

**Comment by tfmorris** [comment]
It looks like this version of the file isn't actually used. So I guess the corresponding updates need to be applied to the live site by an admin.

@mekarpeles should this version be deleted or flagged so people no it isn't used? Or should we update it with the current contents of the production site and then do the edits in git from now on?

**Comment by mekarpeles** [comment]
@tfmorris, that's a good question... 
I think we're creating work for ourselves by managing 2 versions of the file. At the same time, the "production" version of the file isn't necessarily reliable insofar as anyone in dev will not have access to the contents of the production infogami.

I'm honestly not sure all the places where config/edition are used in our code, but it seems reasonable to me to deprecate the version in the git repo and just to edit the production config.

On production, it can be edited here: https://openlibrary.org/config/edition

p.s. @LeadSongDog  + @tfmorris (a) does this sound reasonable and (b) do either of you need write access to this page?

I can update the production values

**Comment by mekarpeles** [comment]
I take it back. It looks like /config/edition is not settable through the `edit` interface. Instead, it apparently requires an admin to submit the document.

That being the case, we should keep the git version around, read it into memory and POST to infogami, if that's how we plan to continue using it.

@gdamdam it looks like you may have been the last to edit this through the the website -- is there anything you can add to our understanding?

**Comment by hornc** [comment]
@mekarpeles https://openlibrary.org/config/edition.yml?m=edit  (as admin) gives a low level, but seemingly sensible, edit page.

**Comment by tfmorris** [comment]
I'm going to start over with the current production data as a base.

**Pull request closed without merging**